plugin.video.dailymotion_com/default.py

```diff
@@ -264,159 +282,172 @@ def listLive(url):
+
+        if len(other_playable_url) > 0:  # probably not needed, only for last resort
             for m_url in other_playable_url:
-                xbmc.log("DAILYMOTION - other m_url = %s" %m_url,xbmc.LOGDEBUG)
-                m_url = m_url.replace('dvr=true&','')
+                xbmc.log("DAILYMOTION - other m_url = {}".format(m_url), xbmc.LOGDEBUG)
+                m_url = m_url.replace('dvr=true&', '')
                 if '.m3u8?sec' in m_url:
-                    rr = requests.get(m_url,cookies=r.cookies.get_dict() ,headers=headers)
+                    rr = requests.get(m_url, cookies=r.cookies.get_dict(), headers=headers)
                     if rr.headers.get('set-cookie'):
                         xbmc.log('DAILYMOTION - adding cookie to url', xbmc.LOGDEBUG)
-                        strurl = re.findall('(http.+)',rr.text)[0].split('#cell')[0]+'|Cookie='+rr.headers['set-cookie']
+                        strurl = re.findall('(http.+)', rr.text)[0].split('#cell')[0] + '|Cookie=' + rr.headers['set-cookie']
                     else:
-                        mburl = re.findall('(http.+)',rr.text)[0].split('#cell')[0]
-                        mb = requests.get(mburl,headers=headers).text
-                        mb = re.findall('NAME="([^"]+)"\n(.+)',mb)
-                        mb = sorted(mb,key=s,reverse=True)
+                        mburl = re.findall('(http.+)', rr.text)[0].split('#cell')[0]
+                        mb = requests.get(mburl, headers=headers).text
+                        mb = re.findall('NAME="([^"]+)"\n(.+)', mb)
+                        mb = sorted(mb, key=s, reverse=True)
                         for quality, strurl in mb:
                             quality = quality.split("@")[0]
                             if int(quality) <= int(maxVideoQuality):
                                 if not strurl.startswith('http'):
-                                    strurl1 = re.findall('(.+/)',mburl)[0]
+                                    strurl1 = re.findall('(.+/)', mburl)[0]
                                     strurl = strurl1 + strurl
                                 break
-                        
-                    xbmc.log("DAILYMOTION - Calculated url = %s" %strurl,xbmc.LOGDEBUG)
+
+                    xbmc.log("DAILYMOTION - Calculated url = {}".format(strurl), xbmc.LOGDEBUG)
                     return strurl
 
+
 def queueVideo(url, name):
     playlist = xbmc.PlayList(xbmc.PLAYLIST_VIDEO)
     listitem = xbmcgui.ListItem(name)
     playlist.add(url, listitem)
 
-def downloadVideo(title,vid):
-    headers = {'User-Agent':'Android'}
+
+def downloadVideo(title, vid):
+    headers = {'User-Agent': 'Android'}
     global downloadDir
     if not downloadDir:
-        xbmc.executebuiltin('XBMC.Notification(Download:,'+translation(30110)+'!,5000)')
-        return    
+        xbmc.executebuiltin('XBMC.Notification(Download:,' + translation(30110) + '!,5000)')
```

Comment:
`xbmcgui.Dialog().notification()` instead of the builtin please. See: https://codedocs.xyz/xbmc/xbmc/group__python___dialog.html#ga27157f98dddb21b44cc6456137977aa2

Reply:
Thanks for spotting, it was in 7 places, have changed all